src/platform-apps/channels/message.tsx

```diff
@@ -175,9 +175,9 @@ export class Message extends React.Component<Properties, State> {
 
   renderMessageWithLinks(): React.ReactElement {
     const { message } = this.props;
-    const hasLinks = linkifyjs.test(message);
+    const hasLinks = linkifyjs.find(message);
 
-    if (hasLinks) {
+    if (hasLinks.length) {
```

Comment:
Just to confirm, `hasLinks` will always be an array - right? This code will break if `hasLinks` is `undefined`/`null` otherwise

Reply:
Yes, `hasLink` will always be an array, the `linkifyjs.find(message)` return always an array.